docs/source/release-notes/workarea-3-4-17.html.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Workarea 3.4.17
+excerpt: Patch release notes for Workarea 3.4.17.
+---
+
+# Workarea 3.4.17
+
+Patch release notes for Workarea 3.4.17.
+
+## Add Order Confirmation Append Point
+
+Add the `storefront.checkout_confirmation_text` append point below the
+`workarea.storefront.checkouts.confirmation_text` copy in storefront.
+
+### Issues
+
+- [76](https://github.com/workarea-commerce/workarea/issues/76)
+
+### Pull Requests
+
+- [76](https://github.com/workarea-commerce/workarea/pull/76)
+
+
+## Display Referrer URL in tooltip on Order Attributes in Admin
+
+Due to the length of URLs being displayed on Order Attributes in the
+admin, they will potentially break layout. Now they are displayed within
+a tooltip behind a "View" link click. The resulting tooltip will prompt
+the user to copy the contents of a text box containing the URL.
+
+### Issues
+
+- [81](https://github.com/workarea-commerce/workarea/issues/81)
+
+### Pull Requests
+
+- [81](https://github.com/workarea-commerce/workarea/pull/81)
+
+## Prevent Error in Product Admin When There Are No Categories
+
+`ProductViewModel#default_category` now protects against a `nil` value
+for the default category before passing its value into a view model.
+This caused an issue for a brand new install when no categories have
+been added yet.
+
+### Issues
+
+- [33](https://github.com/workarea-commerce/workarea/issues/33)
+
+### Pull Requests
+
+- [55](https://github.com/workarea-commerce/workarea/pull/55)
+
+## Improve display of referrer URLs on Orders Show view
+
+Due to the length of URLs being displayed on Order Attributes in the admin
```

Comment:
This is the same description as another item above. Are they duplicates, or should they be combined?